README.md

```diff
@@ -14,7 +14,7 @@
 The [node-mongo-cli](https://code-collabo.gitbook.io/node-mongo/) is a command-line interface made with nodejs. It bootstraps any of these 3 boilerplate templates for your nodejs and/or mongoDB development:
 - [ES module template](https://github.com/code-collabo/node-mongo-esm-kit)
 - [Commonjs template](https://github.com/code-collabo/node-mongo-cjs-kit)
-- Typescript (coming soon)
+- [Typescript](https://github.com/code-collabo/node-mongo-api-boilerplate-template-TS)
```

Comment:
Great job @Olaleye-Blessing. I've seen the other PR too. That one is perfect. 
For this link to TS repo, can you help make a small change? The change is just to make the reader/user know that only manual download option is available for TS. The CLI doesn't generate it yet.
Sorry I didn't think of this earlier.. 

So instead of text for the link just saying "Typescript" just make the text for the link say: "Typescript template (only manual download option available for now)".

Reply:
@Ifycode I don't know if I did the right thing but can you check now?

Reply:
Yes perfect! Thank you.